scripts/data_contamination/README.md

```diff
@@ -65,3 +65,21 @@ with beam.Pipeline() as pipeline:
 ## Notes
 
 The beam script does not support outputting contaminated ngrams yet.
+
+
+## Docker
+
+To create and run docker image:
+
+docker build  . -t contamination_script
+docker run -e INPUT_PATH=<input_path>... SCENARIO_DATA... OUTPUT_STATS... INPUT_FORMAT --rm -it  contamination_script:latest 
+
+where defaults are set as:
+
+ENV INPUT_PATH=input.json
+ENV SCENARIO_PATH=scenario_data
+ENV OUTPUT_PATH=output_stats
+ENV INPUT_FORMAT=the_pile
```

Comment:
Could we not specify the defaults again here to avoid duplication?

Reply:
Sounds good, I removed the ENV variables from the docker file. The question is whether we want the company to just pull the docker image and run it, or build their own image (and then run). If it's the former, I think we'd need ENV variables so they can pass in args. If it's the latter, it's fine to remove

Reply:
Actually I think I misunderstood; we still want the env variables, just no defaults

Reply:
Ok removed the defaults